File: Golang/fibonacci.go
package main

import "fmt"

func fibonacci(n int) int {
	if n == 0 || n == 1 {
		return n
	} else {
		return (fibonacci(n-1) + fibonacci(n-2))
	}
}
func main() {
	var n, i, j int
	j = 0
	fmt.Print("Enter a number: ")
	fmt.Scanln(&n)
	fmt.Print("Fibonacci series: ")
	for i = 1; i <= n; i++ {
		fmt.Print(fibonacci(j), ", ")
		j++
	}
	fmt.Println()
}

File: JS/Fibonacci.js
//Code for Fibonacci series in JavaScript
const nthFibonacci = (n) => {
  // set the base case(fib[0] = 0, fib[1] = 1)
  let fib = [0, 1]
  for (let i = 2; i <= n; i++) {
      fib.push(fib[i - 1] + fib[i - 2]);
  }

  return fib[n];
}
